pipelines/ApVerify.yaml

```diff
@@ -0,0 +1,20 @@
+# Gen 3 pipeline for ap_verify
+# This concatenates various lsst.verify metrics to an AP pipeline
+
+description: Fully instrumented AP pipeline
+inherits:
+    - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+#   - location: $AP_VERIFY_DIR/pipelines/MetricsKpm.yaml  # Currently empty, therefore invalid
+    - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsMisc.yaml
+tasks:
```

Comment:
In writing pipelines I've been either writing meta pipelines with only `inherits` blocks or pipelines with only `tasks` blocks.  I agree, in this case, that having the two mixed makes sense.

Reply:
So are you saying you would create an `ApPipeForApVerify.yaml` that does the config override, then import it in `ApVerify.yaml` in place of the original `ApPipe.yaml`?

How `ap_verify` handles pipelines is likely to get cargo-culted by other AP and/or metrics users, so it's worth thinking about whether it's doing things "right".